apigw-comprehend/example-pattern.json

```diff
@@ -0,0 +1,57 @@
+{
+  "title": "API Gateway to Comprehend",
+  "description": "Create a REST API that acts as a proxy to Comprehend DetectSentiment API operation.",
+  "language": "Python",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This sample project demonstrates how to use an API Gateway REST API as a proxy to Comprehend DetectSentiment API.",
```

Comment:
Amazon API Gateway and Amazon Comprehend

Reply:
Done